### PR TITLE
Upgrade process will not stop due to existing hooks

### PR DESCRIPTION
**Pull request by Prestaworks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Fix for ticket 27184
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes PrestaShop/PrestaShop#27184
| How to test?      | Upgrade from 1.6 to 1.7 succeeds without warnings.
| Possible impacts? | Nothing outside the upgrade process

### CODE REVIEW

File: upgrade/sql/1.7.0.0.sql [hunk 1]
```diff
@@ -154,7 +154,7 @@ CREATE TABLE `PREFIX_module_access` (
 
 /* PHP:add_quick_access_tab(); */;
 
-INSERT INTO `PREFIX_hook` (`name`, `title`, `description`, `position`) VALUES
+INSERT IGNORE INTO `PREFIX_hook` (`name`, `title`, `description`, `position`) VALUES
```
Comment (saulaski) [inline upgrade/sql/1.7.0.0.sql:157]: What about REPLACE to force changes from the upgrade, instead of keeping the previous obsolete values ?

Comment (Prestaworks) [reply upgrade/sql/1.7.0.0.sql:157]: The important thing is the name column, I don't thing we need to write a replace query for this?

**Review by atomiix — DISMISSED** [review]

File: upgrade/sql/1.7.0.0.sql (hunk 1 above)
Comment (saulaski) [reply upgrade/sql/1.7.0.0.sql:157]: Not 100% sure either replacing is important. I just assumed that upgrade could fix a typo in description, or update the position.

Comment (Prestaworks) [reply upgrade/sql/1.7.0.0.sql:157]: I think the position column was related to if the hook allowed drag and drop of the order of the modules.

Comment (saulaski) [reply upgrade/sql/1.7.0.0.sql:157]: Most likely yes, then the command shouldn't replace current position. Your PR is fine :)

**Review by saulaski — APPROVED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by saulaski — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by kpodemski** [comment]
@florine2623 I can confirm this issue. Third-party modules are using these three hooks mentioned in the issue, and if they exist in the DB, the upgrade shows warnings.

**Comment by matks** [comment]
I put back the Waiting for QA again, hoping thanks to @kpodemski we can reproduce the bug

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @Prestaworks

**Pull request merged**